ContosoPizza/Handlers/ReadPizzaHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ContosoPizza.Mediator.Commands.Requests;
+using ContosoPizza.Mediator.Commands.Responses;
+using MediatR;
+
+namespace ContosoPizza.Handlers
+{
+    public class ReadPizzaHandler : IRequestHandler<ReadPizzaRequest, ReadPizzaResponse>
+    {
+        public Task<ReadPizzaResponse> Handle(ReadPizzaRequest request, CancellationToken cancellationToken)
+        {
+            var pizza = PizzaStorage.Pizzas.FirstOrDefault(x => x.Id == request.Id);
```

Comment:
se a pizza for null você vai tomar uma exceção não tratada, testa passando um Id inexistente